meta-lmp-base/recipes-bsp/u-boot/u-boot-base-scr.bb

```diff
@@ -30,6 +30,15 @@ do_deploy() {
     install -m 0644 uEnv.txt ${DEPLOYDIR}
 }
 
+do_install() {
+    install -m 0644 boot.scr ${D}/boot.scr
+    install -m 0644 uEnv.txt ${D}/uEnv.txt
```

Comment:
do we need to have it under root? or is the u-boot script (distro) generic enough to look for content under /boot?

Reply:
AFAIK, distroboot expect boot.scr to be in root of filesystem

Reply:
but yeah, we can move uEnv to `boot/` dir

Reply:
done